### DISCUSSION TITLE
https://docs.avaloniaui.net/docs/guides/data-binding/how-to-create-a-custom-data-binding-converter not foud

### DISCUSSION THREAD

**Discussion by wosiyourfather** [AvaloniaUI/Avalonia#12423] [Q&A]
I need BitmapAssetValueConverter. but the https://docs.avaloniaui.net/docs/guides/data-binding/how-to-create-a-custom-data-binding-converter not foud

**Comment by timunie** [comment]
https://docs.avaloniaui.net/docs/next/guides/data-binding/how-to-create-a-custom-data-binding-converter#example

this is for 11.0

also we have: https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src/Avalonia.Samples/MVVM/ValueConversionSample
